product-info-service/handlers/getProductById.ts

```diff
@@ -0,0 +1,32 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import * as products from '../staticData/productsList.json';
+import 'source-map-support/register';
+import { headers } from '../constants';
+
+export const getProductById: APIGatewayProxyHandler = async (event, _context, _callback) => {
+  if (!event?.pathParameters?.productId) {
+    return {
```

Comment:
It's better throw exception there.
Moreover it better to wrap everuthing in try catch.
Additionally create 2 common hadlers for positiove response and for error